canonical/snapcraft.io
# Update store api version

## Done
- Update store api version
## How to QA
- Go to https://snapcraft-io-4342.demos.haus/login-beta and login
- Go to https://snapcraft-io-4342.demos.haus/admin/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/models
- Click on create model CTA
- Enter a model name but no api-key
- Ensure that you are able to create a model without an api-key
## Issue / Card https://warthogs.atlassian.net/browse/WD-5151
Fixes #

## Screenshots
